pr2eus_moveit/euslisp/robot-moveit.l

```diff
@@ -643,14 +643,26 @@
             (time-step (/ 1 (float (1- size))))
             (time-scale (/ (/ total-time 1000.0) orig-total-time)))
        (dolist (pt points)
-         (when clear-velocities
-           (send pt :velocities #f())
-           (send pt :accelerations #f()))
+         (if clear-velocities
+           (progn
+             (send pt :velocities #f())
+             (send pt :accelerations #f()))
+           (progn
+             (send pt :velocities
+                   (map float-vector #'(lambda (x) (/ x time-scale)) (send pt :velocities)))
+             (send pt :accelerations
+                   (map float-vector #'(lambda (x) (/ x (expt time-scale 2))) (send pt :accelerations)))))
          (send pt :time_from_start (ros::time (+ start-offset-time (* time-scale (send (send pt :time_from_start) :to-sec))))))
        (dolist (pt mdof-points)
-         (when clear-velocities
-           (send pt :velocities nil)
-           (send pt :accelerations nil))
+         (if clear-velocities
+           (progn
+             (send pt :velocities nil)
+             (send pt :accelerations nil))
+           (progn
+             (send pt :velocities
+                   (map float-vector #'(lambda (x) (/ x time-scale)) (send pt :velocities)))
```

Comment:
`(send pt :velocities)` should not be float-vector, but should be `geometry_msgs::Twist` because `pt`'s msg type is `trajectory_msgs::MultiDOFJointTrajectoryPoint`